src/wallet/wallet.h

```diff
@@ -474,7 +474,7 @@ class CWalletObjBase : virtual public MerkleAbstractBase
     virtual void SetNoteData(mapNoteData_t &noteData) {}; // default is null
 
     //! filter decides which addresses will count towards the debit
-    virtual CAmount GetDebit(const isminefilter& filter) const { return 0; } // certs has no vin
+    virtual CAmount GetDebit(const isminefilter& filter) const;
```

Comment:
GetDebit could now be a non-virtual base class method

Reply:
Done